src/components/react-admin/ColorPicker.js

```diff
@@ -40,7 +40,7 @@ const useStyles = makeStyles({
 
 const popover = {
   position: 'absolute',
-  zIndex: '2',
+  zIndex: '11',
```

Comment:
This is a generic component, you shouldn't modify zIndex here

Reply:
Alright. Will modify elsewhere then!

Reply:
Done.